composer.json

```diff
@@ -67,6 +67,9 @@
         ]
     },
     "config": {
+        "platform": {
```

Comment:
this is not really related to platform.sh as we can have 5.6 there .. but I figured we keep things like this for now so that the composer.lock will not cause issues for users still on 5.4